src/containers/SnippetArea.jsx

```diff
@@ -230,7 +229,6 @@ export class SnippetArea extends React.Component {
           canEdit: true,
         }; // Grant all permissions, this is now her file.
         dispatch(setPermissions(permissions));
-        dispatch(updateUserSnippets());
       });
```

Comment:
What else are we able to refactor from this section? Anything that doesn't involve the state of the component or concerns the router could be moved to the then clauses of the thunks